src/PrestaShopBundle/Resources/config/services/adapter/services.yml

```diff
@@ -75,8 +75,13 @@ services:
     class: PrestaShop\PrestaShop\Adapter\Product\PriceFormatter
 
   # Other adapters
+  PrestaShop\PrestaShop\Adapter\Shop\Context:
+    public: false
+
   prestashop.adapter.shop.context:
-    class: PrestaShop\PrestaShop\Adapter\Shop\Context
+    alias: PrestaShop\PrestaShop\Adapter\Shop\Context
+    public: true
```

Comment:
The `public: true` seems not be required here because we set all of services in this file to be public. (line 6)
Anyway, I don't block this PR for this. (It's a deprecated service!)